modules/cli/src/main/scala/scala/cli/ScalaCli.scala

```diff
@@ -45,7 +46,8 @@ object ScalaCli extends CommandsEntryPoint {
   override def description =
     "Scala CLI is a command-line tool to interact with the Scala language. It lets you compile, run, test, and package your Scala code."
   override def summaryDesc =
-    "See 'scala-cli <command> --help' to read about a specific subcommand. To see full help run 'scala-cli <command> --help-full'."
+    """|To run another Scala CLI version, pass '--cli-version' at the beginning of the argument list. 'scala-cli --cli-version <versions> args'.
+       |See 'scala-cli <command> --help' to read about a specific subcommand. To see full help run 'scala-cli <command> --help-full'.""".stripMargin
```

Comment:
I would have left the former message first here, as it should matter to more users. The `--cli-version` stuff should only be relevant for rather advanced users. (I'd be almost reluctant to mention it here actually…)
```suggestion
    """|See 'scala-cli <command> --help' to read about a specific subcommand. To see full help run 'scala-cli <command> --help-full'.
       |To run another Scala CLI version, specify it with '--cli-version' before any other argument, like 'scala-cli --cli-version <version> args'.""".stripMargin
```